analyzer/src/main/kotlin/managers/NPM.kt

```diff
@@ -149,7 +149,12 @@ open class NPM(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConf
             definitionFiles.first().parentFile
         }
 
-        val rootPackageJson = projectRoot.resolve("package.json").readValue<ObjectNode>()
+        val rootDefinitionFile = projectRoot.resolve("package.json")
+        if (!rootDefinitionFile.exists()) {
+            return definitionFiles
```

Comment:
You have to assign some value to the `lateinit var workspaceMatchers`, otherwise `isManagedByWorkspace` will fail.